File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/TransportersUpdateVehiclesRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class TransportersUpdateVehiclesRequestItem {
    @JsonProperty("Id")
    public String id;
    @JsonProperty("LicensePlateNumber")
    public String licensePlateNumber;
    @JsonProperty("Make")
    public String make;
    @JsonProperty("Model")
    public String model;
}

File: sdks/thunkmetrc-go/wrapper/models/transporters_update_vehicles_request_item.go
package models

type TransportersUpdateVehiclesRequestItem struct {
    Id string `json:"Id,omitempty"`
    LicensePlateNumber string `json:"LicensePlateNumber,omitempty"`
    Make string `json:"Make,omitempty"`
    Model string `json:"Model,omitempty"`
}
